src/test/java/io/odh/test/e2e/standard/UninstallST.java

```diff
@@ -122,4 +144,8 @@ void deployDataScienceCluster() {
         ResourceManager.getInstance().createResourceWithWait(dsci);
         ResourceManager.getInstance().createResourceWithWait(dsc);
     }
+
+    static boolean isOdhTested() {
+        return Environment.PRODUCT.equalsIgnoreCase(Environment.PRODUCT_ODH);
```

Comment:
This should check for OLM, not ODH, I think, like I had in my PR

https://github.com/skodjob/odh-e2e/pull/116/files#diff-112f35283ce54c71303c8ab1798ae4fabd4b8660eadc4ddea3073c1fa1c593b1R128-R130

Reply:
This is intentional - I disabled this test on ODH for now.